chrome/stable/Dockerfile

```diff
@@ -6,13 +6,14 @@
 #	--memory 512mb \ # max memory it can use
 #	-v /tmp/.X11-unix:/tmp/.X11-unix \ # mount the X11 socket
 #	-e DISPLAY=unix$DISPLAY \
-#	-v $HOME/Downloads:/root/Downloads \
+#	-v $HOME/Downloads:/home/chrome/Downloads \
 #	-v $HOME/.config/google-chrome/:/data \ # if you want to save state
 #	--device /dev/snd \ # so we have sound
 #	-v /dev/shm:/dev/shm \
 #	--name chrome \
 #	jess/chrome
 #
+# For Ubuntu I had to add --privileged to above `docker run` command
```

Comment:
this is because you need a different seccomp policy it lives here: https://github.com/jessfraz/dotfiles/blob/master/etc/docker/seccomp/chrome.json I would just put that in the comment instead, since I wouldnt want people to run it as priviledged